assets/js/base/context/providers/store-notices/components/store-notices-container.js

```diff
@@ -24,8 +23,7 @@ const getWooClassName = ( { status = 'default' } ) => {
 	return '';
 };
 
-const StoreNoticesContainer = ( { className, notices } ) => {
-	const { removeNotice } = useStoreNoticesContext();
+const StoreNoticesContainer = ( { className, notices, removeNotice } ) => {
```

Comment:
🤔 was the primary reason for this change to remove the need for the context dependency? If so 👏 - with this change it does mean this component doesn't _have_ to live in the context folder tree but it probably still makes sense for it to be here for now.

Reply:
yup, wasn't necessary as it's easy enough to pass down.